src/components/Composer/index.js

```diff
@@ -251,20 +250,34 @@ class Composer extends React.Component {
         } catch (e) {}
     }
 
+    /**
+     * Manually place the pasted HTML into Composer
+     *
+     * @param {String} html - pasted HTML
+     */
+    handlePastedHTML(html) {
+        const parser = new ExpensiMark();
+
+        this.paste(parser.htmlToMarkdown(html));
+    }
+
     /**
      * Check the paste event for an attachment, parse the data and call onPasteFile from props with the selected file,
      * Otherwise, convert pasted HTML to Markdown and set it on the composer.
      *
      * @param {ClipboardEvent} event
      */
     handlePaste(event) {
+        if (event) {
+            event.preventDefault();
```

Comment:
Any specific reason why we added this?

Reply:
I believe `preventDefault` is used to cancel the regular paste. Previously we also use `preventDefault` on two block conditions (pasting files and pasting HTML text), since we now format all of the pasted content I just move the `preventDefault` out of the condition 'cause we only need that in all scenarios.